### ISSUE TITLE
Rethink resources section of site

### ISSUE THREAD

**Issue by hagbardc** [IndivisibleOnline/Indivisible-Plugin#22]
Rethink resources section of site

**Comment by imlindaryan** [comment]
Agreed.  This needs a meeting with Communications and Coaching.  Propose set an internal conversation for week of 6/10
Will take on task of looping in Comms
